src/Carousel.svelte

```diff
@@ -104,7 +104,7 @@
 	onMount(() => {
 		controller = new Siema({
 			selector: siema,
-			perPage,
+			perPage: isNaN(perPage)? perPage: Number(perPage),
```

Comment:
What's the purpose of checking `perPage` using `isNaN`?

I feel like this is defensive code when, if people try to pass a non-number, then we should just let the code error.

For instance, what if they pass a string like `foo`? We just want to try to parse it and throw, right?

Reply:
Well, they can pass a responsive object({
  perPage: {
    800: 2, // 2 items for viewport wider than 800px
    1240: 3 // 3 items for viewport wider than 1240px
  })
, and should not fail, so it seemed the more straightforward to let both cases being handled.  
 other way could be check if is an string and in that case use parseInt.

Really the original problem comes from html attributes, that are string by default, if instead of setting the perPage directly, you define a number variable in javascript and bind the prop to that variable it works also correctly.

Reply:
Ah right.

So I think we should probably handle this by saying `typeof perPage === 'object' ? perPage : Number(perPage)` - what do you reckon?

Reply:
Yeah, no problem, I will update the pull request